backend/api/src/model/series.rs

```diff
@@ -0,0 +1,54 @@
+use juniper::{graphql_object, FieldResult};
+
+use crate::{Context, id::Key, Id, model::event::Event, util::RowExt};
+
+
+pub(crate) struct Series {
+    key: Key,
+    title: String,
+    description: Option<String>,
+}
+
+#[graphql_object(Context = Context)]
+impl Series {
+    fn id(&self) -> Id {
+        Id::series(self.key)
+    }
+
+    fn title(&self) -> &str {
+        &self.title
+    }
+
+    fn description(&self) -> Option<&str> {
+        self.description.as_deref()
+    }
+
+    async fn events(&self, context: &Context) -> FieldResult<Vec<Event>> {
+        Event::load_for_series(self.key, context).await
+    }
+}
+
+impl Series {
+    pub(crate) async fn load_by_id(id: Id, context: &Context) -> FieldResult<Option<Self>> {
+        let result = if let Some(key) = id.key_for(Id::SERIES_KIND) {
+            context.db.get()
+                .await?
+                .query_opt(
+                    "select id, title, description
+                        from series
+                        where id = $1",
+                    &[&(key as i64) as _],
```

Comment:
```suggestion
                    &[&(key as i64)],
```

This works apparently